launch/docker.go

```diff
@@ -0,0 +1,90 @@
+package launch
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+)
+
+type docker struct {
+	volume            string
+	setupImage        string
+	setupImageVersion string
+}
+
+var _ runner = (*docker)(nil)
+var execCommand = exec.Command
+
+const (
+	ArtifactsDir = "artifacts"
+	LogFile      = "builds.log"
+	// The definition of "ScmHost" and "OrgRepo" is in "PipelineFromID" of "screwdriver/screwdriver_local.go"
+	ScmHost = "screwdriver.cd"
+	OrgRepo = "sd-local/local-build"
+)
+
+func newDocker(setupImage, setupImageVer string) runner {
+	return &docker{
+		volume:            "SD_LAUNCH_BIN",
+		setupImage:        setupImage,
+		setupImageVersion: setupImageVer,
+	}
+}
+
+func (d *docker) setupBin() error {
+	_ = execCommand("docker", "volume", "rm", "--force", d.volume).Run()
```

Comment:
Don't we need to check this error?

Reply:
Since I am running with `--force`, I think that there will be almost no errors other than the bellow errors.
We also believe that there is no problem in ignoring the following errors.

Error trying to delete "volume" when it doesn't exist.

Reply:
Make sense. 
Is it better just call
```
execCommand("docker", "volume", "rm", "--force", d.volume).Run()
```

Reply:
If there is no left side, the editor warned.
So I did this.

Reply:
I see